submissions/YuraZagor/hooliPopup/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+	box-sizing: border-box;
+	position: relative;
+	margin: 0;
+	padding: 0;
+}
+
+
+.ul--menu, .popups--container, .popups--container--all {
+	list-style-type: none;
+	
+}
+
+.ul--menu {
+	display:flex;
+	justify-content: flex-end;
+	align-items: center;
+	
+}
+.li--menu {
+	padding: 10px;
+	
+}
+
+.li--menu img {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)